code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -711,3 +711,64 @@
 	item_state = "syndie_helm"
 	item_color = "syndi"
 	armor = list("melee" = 40, "bullet" = 50, "laser" = 30, "energy" = 15, "bomb" = 35, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
+
+	// Captain's shiny special hardsuit
+/obj/item/clothing/head/helmet/space/hardsuit/captain
+	name = "captain's hardsuit helmet"
+	icon_state = "hardsuit0-captain"
+	item_state = "capspacehelmet"
+	item_color = "captain"
+	desc = "A special helmet designed for only the most fashionable of military figureheads."
+	flags_inv = HIDEFACE
+	permeability_coefficient = 0.01
+	armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
```

Comment:
Oh, and this too actually:
![image](https://user-images.githubusercontent.com/57483089/127937720-40b69c3f-f213-4816-b24a-608465752352.png)

```suggestion
	flags_inv = HIDEFACE
	armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
```